qa/tasks/radosgw_admin.py

```diff
@@ -918,6 +918,12 @@ def task(ctx, config):
 
     (err, out) = rgwadmin(ctx, client, zonecmd, check_status=True)
 
+    # TESTCASE 'zone-info', 'zone', 'get', 'get zone info', 'succeeds'
```

Comment:
you don't need to add a new test for the default, just remove --rgw-zone from the old zone get test